src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java

```diff
@@ -28,14 +31,53 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
             playerMsg(TFM_Command.PLAYER_NOT_FOUND);
             return true;
         }
-
-        smite(player);
+        else if (args.length > 1)
+        {
+            final String reason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
+            smite(player, reason);
+            return true;
+        }
+        else
+        {
+            smite(player);
+        }
 
         return true;
     }
 
+    public static void smite(final Player player, final String reason)
+    {
+        // Smites the player with the reason under it in yellow
+        TFM_Util.bcastMsg(String.format("%s has been a naughty, naughty boy. \n" + ChatColor.YELLOW + "Reason: %s", player.getName(), reason), ChatColor.RED);
+
+        //Deop
+        player.setOp(false);
+
+        //Set gamemode to survival:
+        player.setGameMode(GameMode.SURVIVAL);
+
+        //Clear inventory:
+        player.getInventory().clear();
```

Comment:
@Wild1145 If you look at /smite's source, you can see that clearing inventories has always been a thing.